kdl-client/src/Components/CircledInfoMessage/CircledInfoMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styles from './CircledInfoMessage.module.scss';
+import cx from 'classnames';
+
+export enum CircledInfoMessageTypes {
+  SUCCESS,
+  ERROR,
+}
+
+type Props = {
+  type: CircledInfoMessageTypes;
+  text?: string;
+  className?: string;
+};
+
+const CircledInfoMessage: React.FC<Props> = ({
+  type = CircledInfoMessageTypes.SUCCESS,
+  text = '',
+  children,
+}) => {
+  return (
+    <div
+      className={cx(styles.circledMessage, {
```

Comment:
with previous change, you can now change this:
```suggestion
      className={cx(styles.circledMessage, styles[type])}
```
and remove lines 25 and 26. This way is easier to make updates when adding more types